src/test/Components/CreateImageWizard/steps/Compliance/Compliance.test.tsx

```diff
@@ -0,0 +1,157 @@
+import { screen, waitFor, within } from '@testing-library/react';
+import { userEvent } from '@testing-library/user-event';
+
+import { CREATE_BLUEPRINT, EDIT_BLUEPRINT } from '../../../../../constants';
+import { CreateBlueprintRequest } from '../../../../../store/imageBuilderApi';
+import { mockBlueprintIds } from '../../../../fixtures/blueprints';
+import { complianceCreateBlueprintRequest } from '../../../../fixtures/editMode';
+import { clickNext } from '../../wizardTestUtils';
+import {
+  clickRegisterLater,
+  enterBlueprintName,
+  interceptBlueprintRequest,
+  interceptEditBlueprintRequest,
+  openAndDismissSaveAndBuildModal,
+  renderCreateMode,
+  renderEditMode,
+} from '../../wizardTestUtils';
+
+// Overwrite
+vi.mock('@unleash/proxy-client-react', () => ({
+  useUnleashContext: () => vi.fn(),
+  useFlag: vi.fn((flag) => {
+    switch (flag) {
+      case 'image-builder.compliance.enabled':
+        return true;
+      default:
+        return false;
+    }
+  }),
+}));
+
+const goToComplianceStep = async () => {
+  const user = userEvent.setup();
+  const guestImageCheckBox = await screen.findByRole('checkbox', {
+    name: /virtualization guest image checkbox/i,
+  });
+  await waitFor(() => user.click(guestImageCheckBox));
+  await clickNext(); // Registration
+  await clickRegisterLater();
+  await clickNext(); // Compliance
+  await screen.findByRole('heading', { name: /Compliance/ });
+};
+
+const goToReviewStep = async () => {
+  await clickNext(); // File system configuration
+  await clickNext(); // Custom repositories
+  await clickNext(); // Additional packages
+  await clickNext(); // Details
+  await enterBlueprintName('Compliance test');
+  await clickNext(); // Review
+};
+
+const selectPolicy = async () => {
+  const user = userEvent.setup();
+  const policyMenu = await screen.findByRole('button', {
+    name: /options menu/i,
+  });
+  await user.click(policyMenu);
+
+  const cisl2 = await screen.findByRole('option', {
+    name: /CIS Red Hat Enterprise Linux 8 Benchmark for Level 2 - Workstation/i,
+  });
+  await user.click(cisl2);
```

Comment:
This should solve the `was not wrapped in act(...)` hell in test output

```suggestion
  await waitFor(() => user.click(policyMenu));

  const cisl2 = await screen.findByRole('option', {
    name: /CIS Red Hat Enterprise Linux 8 Benchmark for Level 2 - Workstation/i,
  });
  await waitFor(() => user.click(cisl2));
```